pkg/operatingsystem/cos/kernel-package.go

```diff
@@ -0,0 +1,269 @@
+package cos
+
+import (
+	"archive/tar"
+	"bufio"
+	"compress/gzip"
+	"encoding/base64"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/thought-machine/falco-probes/pkg/docker"
+	"github.com/thought-machine/falco-probes/pkg/operatingsystem"
+)
+
+const (
+	image                      = "docker.io/library/falco-driver-loader:0.33.0"
```

Comment:
Does this mean we will only build probes for Falco `0.33.0` and update this value as new versions are released?

Is there a way we can build probes for different versions of Falco, including old versions we don't still use and new versions we don't yet use, given this is an open source project that may be used by other people?

Reply:
Very good point thanks! I'll dig into this a bit - from https://github.com/thought-machine/falco-probes/pull/51 it looks like we already list versions so it may just be a case of using a reusing variable from somewher 🤞

Reply:
Looks like probe build is done by [BuildEBPFProbe](https://github.com/thought-machine/falco-probes/blob/628cf5548a2b26505b879391b2cc7bcbc5e59990/pkg/falcodriverbuilder/build-ebpf-probe.go#L14) which, 

1. takes a list of falco versions and the OS,
2. loads the OS data (kernel config, os-release vars) and
3. for each version builds the probe on _that version_.

So the choice of image highlighted is fairly arbitrary, I'll probably change this to something smaller like [busybox, as used by volume.go](https://github.com/thought-machine/falco-probes/blob/628cf5548a2b26505b879391b2cc7bcbc5e59990/pkg/docker/volume.go#L17-L21).